odoo/modules/registry.py

```diff
@@ -332,6 +332,9 @@ def post_init(self, func, *args, **kwargs):
 
     def post_constraint(self, func, *args, **kwargs):
         """ Call the given function, and delay it if it fails during an upgrade. """
+        if kwargs.pop("postpone", False):
+            self._constraint_queue.append(partial(func, *args, **kwargs))
+            return
```

Comment:
Error log should be removed by adt in his own pr and then this hack could be removed.

Reply:
Well, the tests pass without this hack being used, so you can remove it.

Reply:
I've added a "test" to highlight the problem, the error log is still there in the runbot ;)

Reply:
@Feyensv I just pushed a commit that fixes the source of the problem.